modules/widespaceBidAdapter.js

```diff
@@ -177,7 +177,10 @@ export const spec = {
 function storeData(data, name, stringify = true) {
   const value = stringify ? JSON.stringify(data) : data;
   if (LOCAL_STORAGE_AVAILABLE) {
-    localStorage.setItem(name, value);
+    try {
```

Comment:
Lets do this in separate PR and tag widespace contributors so they are aware of this change

Reply:
ok, I will revert this change

Reply:
Reverted the commit.